modules/rank-eval/src/test/java/org/elasticsearch/index/rankeval/RatedRequestsTests.java

```diff
@@ -122,6 +122,7 @@ public void testXContentRoundtrip() throws IOException {
         }
     }
 
+    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/104570")
```

Comment:
Is there a way to only mute for Windows? I think it only happens there?

Reply:
Yes, like this: https://github.com/elastic/elasticsearch/blob/2bc739875465d2e3048e51a81a5d973537c6c460/x-pack/plugin/ml/qa/native-multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/ForecastIT.java#L225

Reply:
Thanks, fixed